integration-tests/capabilities/components/evmcontracts/capabilities_registry/capabilities_registry.go

```diff
@@ -0,0 +1,70 @@
+package capabilities_registry
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/seth"
+	cr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+)
+
+type CapabilitiesRegistryInstance struct {
+	Address                       common.Address
+	Contract                      *cr.CapabilitiesRegistry
+	sc                            *seth.Client
+	ExistingHashedCapabilitiesIDs [][32]byte
+}
+
+func Deploy(sc *seth.Client) (*CapabilitiesRegistryInstance, error) {
```

Comment:
can we not reuse, extend and unify on the work in deployment/keystone?

ccip is doing something similiar -- all deployment logic is unified and they have a `devenv` for integration tests